setup.py

```diff
@@ -200,9 +203,37 @@ def build_extensions(self):
         return super().build_extensions()
 
 
-cmdclass = versioneer.get_cmdclass()
+def update_matplotlibrc(path):
+    # Update the matplotlibrc file if packagers want to change the default
+    # backend.
+    template_lines = path.read_text().splitlines(True)
+    backend_line_idx, = [  # Also asserts that there is a single such line.
+        idx for idx, line in enumerate(template_lines)
+        if line.startswith("#backend:")]
+    if setupext.options["backend"]:
```

Comment:
Can we get into trouble if this had been set once but has been removed again? In that case we would not revert to the original value.

Reply:
indeed